### DISCUSSION TITLE
Has anyone tried this with an IPS Flow Cell?

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#882] [Q&A] [answered]
Looking forward to phase 2 of my project--using REM Chem with pH & ORP probes to drive the chlorine generator (salt cell) & acid pump...

I have concerns about adapting the IPS flow cell to use the Atlas Scientific industrial probes--it's way beyond my skill set.

Has anyone tried using adapters like this instead of enlarging the hole & tapping it as described in the wiki?  I don't know if the adapter is too long to allow the probe to be embedded in the water flow.

https://www.amazon.com/SUNGATOR-Fittings-Reducer-Adapter-Fitting/dp/B0B8TZTTTK/ref=m_crc_dp_lf_d_t1_d_sccl_2_6/146-3148965-4336464?pd_rd_w=oBfZZ&content-id=amzn1.sym.76a0b561-a7b4-41dc-9467-a85a2fa27c1c&pf_rd_p=76a0b561-a7b4-41dc-9467-a85a2fa27c1c&pf_rd_r=GMW0862QJYES2H6T9RPA&pd_rd_wg=J5sLS&pd_rd_r=58a39d95-90ba-4aab-9493-12c76466ad9c&pd_rd_i=B0B8TZTTTK&th=1

Thanks for your thoughts!

**Comment by tagyoureit** [comment, marked as answer]
It doesn't look like the industrial probe will fit inside the 1/2" portion of the adapter.  @rstrouse helped me drill into the ips switch and if you go slow and steady it works well.  

![image](https://github.com/tagyoureit/nodejs-poolController/assets/1646840/8250aac1-fb6d-428a-94eb-adc839d6866b)

![image](https://github.com/tagyoureit/nodejs-poolController/assets/1646840/064f9c71-bdad-432a-9149-b35e0c9454a0)

**Reply by FlaMike** [reply]
Shoot!  That's a darned shame!!!  Thanks for the info on the probe dimensions & saving me *that* hassle.  I'm guessing the adapter's dimensions are typical.

Hey, @rstrouse could you be cajoled, bribed, er paid to drill & tap an IPS Flow Cell?  I'm serious, I'm willing to make it worth your time...

**Reply by rstrouse** [reply]
Do you have a drill press?  If you do this is really simple.  All you need to do is buy a 3/4 npt tap and bit.  Then put the but in your drill press with the thing unplugged.   That way you don't have any of the goofball runout issues that a cheap drill press has.   Then build the jig in the instructions and screw the flow cell to the two pieces of wood.   Clamp that to the drill press so that the bit alignment is offset just a bit.  Then using the power of Stretch Armstrong, hand twist the chuck to drill out the hole.  It will come out perfect every time, it will take minutes, and you will be shocked that it came out so good.

After you have the hole it is just a matter of eyeballing the tap into the hole.  Run it in to leave about 7 or 8 threads at the top.  For each 1/2 turn on the tap back it out at least a 1/4 turn to clean out the threads.  Once you are done you will revel in your handiness.

**Reply by FlaMike** [reply]
I think I can bribe someone with a drill press to let me use is unpowered as you suggest.  Can't wait to enlist the power of "Stretch Armstrong"  then marvel at the results.  Your suggestions go beyond the wiki and are appreciated as always.

**Reply by FlaMike** [reply]
@tagyoureit & @rstrouse I wanted to let you know I found someone willing to let me use his drill press & with his help this morning I successfully enlarged & offset the holes and tapped the threads in the IPS flow cell.  Per @rstrouse 's suggestion above, the drilling was  done without turning on the motor & my friend & I supplied the "horsepower."  Definitely, less than 1/4 HP.  :-)

I'm now waiting for the ORP & pH probes with the SMA connectors to become available again at Atlas Scientific.  Seems they're out of stock at the moment. 

Do I need to change my username from FlaMike to FlaStretch?

Thanks for your support & suggestions!

**Reply by rstrouse** [reply]
At least you didn’t have to break out the GI Joe with the kung fu grip.   It turned out to be totally doable you didn’t need to stress over it.  Good job!
